app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/DraftExciseMovementController.scala

```diff
@@ -80,6 +79,22 @@ class DraftExciseMovementController @Inject()(
         result.merge
     }
 
+
+
+  }
+
+  private def ssa_internal(request: ParsedXmlRequest[_], ern: String, boxId: Option[String], message: IE815Message)(implicit hc: HeaderCarrier): EitherT[Future, Result, Movement] = {
+    EitherT {
+      submissionMessageService.submit(request, ern).flatMap {
```

Comment:
If the future fails, we won't audit anything... is this what's intended?

Reply:
Also.... this submit method also has calls to NRS in them... how do they relate?

Reply:
What if the submission to the backend succeeds, but the call to NRS doesn't... could we end up with no audit on a successful event?

Reply:
@Froom2 are your comments here still needing to be looked at?